<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3FixitConfig</class>
 <widget class="QDialog" name="Q3FixitConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q3Fixit - Config</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Q3 Config fixar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="FolderText">
       <property name="placeholderText">
        <string>Välj quake3.exe</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="BrowseBtn">
       <property name="text">
        <string>Bläddra</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="RunBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Updatera Q3config!</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignTop">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Q3 Config</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="0" colspan="2">
         <widget class="QCheckBox" name="ModelChk">
          <property name="text">
           <string>Gör mig snygg, ange modell:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="LanChk">
          <property name="text">
           <string>Inget lagg tack (LAN-style)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="BrightnessChk">
          <property name="text">
           <string>Gör det ljust och fint</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2" alignment="Qt::AlignLeft">
         <widget class="QComboBox" name="ModelCombo"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="MemoryChk">
          <property name="text">
           <string>Ge qvack lite minne så det flyter (hunken)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ResolutionChk">
          <property name="text">
           <string>Fixa upplösningen så det blir krispigt och fint</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="FPSChk">
          <property name="text">
           <string>Get mig optimala fps bitte (maxfps, ..)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
